archive/walk.go

```diff
@@ -273,110 +224,91 @@ func (l LogID) Path(ark *Archive) iosrc.URI {
 	return ark.DataPath.AppendPath(string(l))
 }
 
-type SpanInfo struct {
-	First       nano.Ts // timestamp of first record in this span
-	Last        nano.Ts // timestamp of last record in this span
-	LogID       LogID
-	RecordCount int64
+type Chunk struct {
```

Comment:
This can wait for another PR, but a Chunk represents a data file, right? So why not call it DataFile?

Reply:
Interesting question. A Chunk represents both the data & seek index file, would also hold information about the files it 'masks' in the future. I'll leave as-is for this PR, but can discuss more.